src/remix/RemixClient.ts

```diff
@@ -121,7 +125,7 @@ export class RemixClient {
         try{
             response = await axios.get(`${SERVER_URL}/files/${chain}/${address}`)
         } catch(err) {
-            response = err.response;
+            throw new Error(err)
```

Comment:
This is the block I was referring to in the earlier comment. So you claim that just the `response = await axios.get...` without an immediate catch produces an unhandled error? I mean, this way it doesn't seem wrong, just redundant.

Reply:
should be ok bow